spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -33,11 +33,11 @@ To meet those requirements, we introduce 2 new claims to the MP-JWT:
 
 * "upn": A human readable claim that uniquely identifies the subject or user principal of the token, across
 the MicroProfile services the token will be accessed with.
-* "groups": The token subject's group memberships that will be mapped to Java EE style application
-level roles in the MicroProfile service container.
+* "groups": A list of group names to be assigned to the principal of the MP-JWT. This typically will require a mapping at the application container level to application roles, but a one-to-one between group names and application role names is required to be performed in addition to any other mapping when this claim is available in the token.
```

Comment:
Last part can be confusing because we basically say "Some other mapping is **required** on the group names". but we do not specify what that means. But use the word **required**.

I guess we want to leave the option open that the runtime performs any custom mapping on the group names which are configured (implementation specific)

Reply:
@rdebusscher I've added a note below that the custom mapping is possible. The above just says that of no mapping is done then the groups values are taken as is.

Reply:
I still find it confusing (the one-to-one mapping AND any other mapping is required). But everyone else if fine with this, you can merge it.

Proposal 

>This typically will require a mapping at the application container level to application roles, a one-to-one between group names and application role names is the default when no custom mapping is defined in an implementation specific way.